CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [5.0.0] - 2022-05-10
+### Added
+- Add retry logic to `getBlocksForRange`, treating a null block number as failure ([#61](https://github.com/MetaMask/eth-json-rpc-filters/pull/61))
+
+### Changed
+- **BREAKING:** Drop support for Node 10 ([#50](https://github.com/MetaMask/eth-json-rpc-filters/pull/50))
```

Comment:
We're dropping support for all versions lower than 10 as well. There was no minimum version until this release

```suggestion
- **BREAKING:** Set minimum Node.js version to v12 ([#50](https://github.com/MetaMask/eth-json-rpc-filters/pull/50))
```

Reply:
Ah, yeah, good point.